app/assets/javascripts/sync.coffee

```diff
@@ -126,8 +149,8 @@ class Sync.Pusher extends Sync.Adapter
     !!window.Pusher
 
   connect: ->
-    opts =
-      encrypted: SyncConfig.pusher_encrypted
+    # Pusher doesn't properly transform native boolean flags in options.
+    opts = { encrypted: SyncConfig.encryption_flag.toString() }
```

Comment:
Yeah, that was a weird one. Although, somebody will have to keep an eye on that (re: pusher). I'm not using it anymore, as we found a less expensive option without faye's issues, rabbitmq + stomp. They may correct this in the future.